tests/unit/tasks/mocks.py

```diff
@@ -0,0 +1,7 @@
+import htetl.tasks.sites as tasks_sites
+from htetl.tasks.loadpages import RawPageData
+
+
+def setup_RawPageData():
```

Comment:
Isn't a `pass` needed here?

Reply:
Honestly, I don't know why this file is here.